### PR TITLE
feat: add debugger to the host hook to simplify the inspect

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

**Comment by Jack-Works** [comment]
oh, it seems like I have requested this feature month ago... @devsnek hi

File: src/worker.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ addEventListener('message', ({ data }) => {
 
     const agent = new Agent({
       features: [...state.get('features')],
+      onDebugger() { debugger }, // eslint-disable-line no-debugger
```
Comment (devsnek) [inline src/worker.js:30]:
```suggestion
      onDebugger() {
        debugger; // eslint-disable-line no-debugger
      },
```

**Comment by Jack-Works** [comment]
Oh I have deleted forked repo in last few months so if I have to commit the change, I need to re-fork and open a new pr. Do you accept this without review resolved? Or I need to do it later

**Comment by devsnek** [comment]
pushed manually.

**Comment by Jack-Works** [comment]
Cool thanks!

**Pull request closed without merging**